api_example/postit_api/serializers.py

```diff
@@ -7,15 +7,15 @@
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ['username', 'password']
+        fields = ['username', 'password', 'first_name', 'last_name', 'email']
         extra_kwargs = {'password': {'write_only': True}}
 
-    def create(self, validated_data):
-        password = validated_data.pop('password')
-        user = User(**validated_data)
-        user.set_password(password)
-        user.save()
-        return user
+        def create(self, validated_data):
```

Comment:
* return `def create` back to root class structure, not under `class Meta`
* create method `def update` and
    * pop password from kwargs, and if it was changed, do `user.set_password(password)`